pkgs/development/python-modules/tinycss2/default.nix

```
@@ -12,7 +12,14 @@ buildPythonPackage rec {
     sha256 = "1kw84y09lggji4krkc58jyhsfj31w8npwhznr7lf19d0zbix09v4";
   };
 
-  patches = [ ./remove-redundant-dependency.patch ];
+  patches = [
+    ./remove-redundant-dependency.patch
+    # remove this patch on next version bump
```

I usually like to know why a patch is being applied. You already did the other half (saying when it's ready to remove) :+1: 
```suggestion
    # fixes tests, remove this patch on next version bump
```

---

Fixed!